hubspot_sync/management/commands/configure_hubspot_properties.py

```diff
@@ -637,26 +638,88 @@
 }
 
 
-def upsert_custom_properties():
+def _get_course_run_certificate_hubspot_property():
+    """
+    Creates a dictionary representation of a Hubspot checkbox,
+    populated with options using the string representation of all course runs.
```

Comment:
does "string representation" mean "readable id" ?

I'm not sure this is the best place to put it, but an example might help.

Reply:
For course runs, this would be `courseware_id | title`.

Reply:
Can you give an example? Based on `courseware_id | title` I'm hoping an example would be "course-v1:MITx+14.73x+1T204 | The Challenges of World Poverty"

Reply:
course-v1:MITxT+14.100x+3T2023 | Microeconomics